templates/quickstart-bitbucket-dc.template.yaml

```diff
@@ -628,131 +672,110 @@ Resources:
     Type: AWS::AutoScaling::LaunchConfiguration
     Metadata:
       AWS::CloudFormation::Init:
-        configSets:
-          default: [1, 2]
-        '1':
-          packages:
-            !If
-            - SetupCollectd
-            - yum:
-                collectd: []
-                collectd-java: []
-                collectd-generic-jmx: []
-                collectd-rrdtool: []
-            - !Ref AWS::NoValue
+        # configSets:
```

Comment:
Any reasons why you've left commented sections?

Reply:
Missed them during clean-up; removed.